v2.1/admin-ui-statements-page.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Statements Page
```

Comment:
Please add a `summary:` field to the front-matter. That's used as the page's meta description for SEO.